### ISSUE TITLE
Jakarta EE 10 / Quarkus 3

### ISSUE THREAD

**Issue by snazy** [projectnessie/nessie#5264]
Quarkus 3 will move to Jakarta EE 10. This means that basically all `import javax.*` have to be replaced with `import jakarta.*`. All the `jakarta.*` packages are built for bytecode version 55 (Java 11), which means, that the dependencies cannot "just be bumped with some find+replace".

Hint:
```
jakarta/servlet/http/HttpServletRequest has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0
```

As long as only annotations are affected, I think, that it's possible (although definitely not really nice) to _repeat_ the annotations for `javax.*` and `jakarta.*`. For example instead of having
```java
  @NotBlank
  public abstract String getMessage();
```
use:
```java
  @javax.validation.constraints.NotBlank
  @jakarta.validation.constraints.NotBlank
  public abstract String getMessage();
```

 See #5255 for dependencies.

**Comment by snazy** [comment]
Something to keep in mind:
```
jakarta/servlet/http/HttpServletRequest has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0
```
Translated: Jakarta APIs don't work with Java 8

**Comment by snazy** [comment]
Fixed via #6172 and #6174
